client/components/tinymce/plugins/insert-menu/style.scss

```diff
@@ -0,0 +1,31 @@
+.mce-container .mce-wpcom-insert-menu {
+	& .mce-caret {
+		padding-left: 12px;
+	}
+}
+
+.mce-container.mce-menu .mce-container-body .mce-wpcom-insert-menu__menu-item {
+	margin-left: -6px;
+
+	& svg {
+		fill: $gray;
+	}
+
+	& span {
```

Comment:
Adding to @mtias note above, this will break in React 15.0: https://facebook.github.io/react/blog/2016/04/07/react-v15.html#no-more-extra-ltspangts